### PR TITLE
RTD Fixes

### PR DESCRIPTION
**Pull request by llyon-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by llyon-cb** [comment]
LMK what you think about these changes @avanbrunt-cb

File: docs/index.rst [hunk 1]
```diff
@@ -39,10 +39,10 @@ which is explained in the `Developer Network Authentication Guide
 The SDK manages your API credentials for you. There are multiple ways to supply the SDK
 with your API credentials, which is explained in :ref:`authentication`.
 
-User Guide
-----------
+Get Started
```
Comment (avanbrunt-cb) [inline docs/index.rst:42]: This reads a little strange maybe `Getting Started`?

Comment (llyon-cb) [reply docs/index.rst:42]: Yeah that works for me

**Review by avanbrunt-cb — APPROVED** [review]
One minor otherwise looks good and thanks for the additional deep review

**Pull request merged**